mfa/mfa.go

```diff
@@ -175,6 +175,12 @@ func (m *Service) handleRequest(ctx context.Context, decodedJWT entities.JWTData
 	claims, _ := m.generateClaims(requestFlow, decodedJWT, &challenge)
 	token, _ := m.JWTService.GenerateToken(claims, scopes)
 
+	if err != nil {
```

Comment:
Since we are dealing with `err` variable in this PR, maybe we can name it something a bit more explicit to distinguish from other errors in the same function, something like `result, errMakingReq := requestFlow.Requ...`